db/migrate/20160518004516_add_peer_review_to_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPeerReviewToAssignments < ActiveRecord::Migration
+  def change
+    add_column :assignments, :peer_review, :boolean, default: false
```

Comment:
Again, I don't want to re-use the name "peer_review" too much. Maybe change the column name to "has_peer_review". Also, put in a null: false.